br/pkg/task/restore_raw.go

```diff
@@ -137,6 +146,11 @@ func RunRestoreRaw(c context.Context, g glue.Glue, cmdName string, cfg *RestoreR
 	updateCh.Close()
 
 	if cfg.Checksum {
+		if !featureGate.IsEnabled(feature.Checksum) {
```

Comment:
I think it would be better to reject restore if user requires checksum but TiKV does not support, and prompt that checksum must be disable.
Most users just ignore the warning in logs if command succeed.

Reply:
Ok, agreed. PTAL~